adapter/src/dbt/adapters/fal_experimental/connections.py

```diff
@@ -43,8 +43,11 @@ def cancel(self, connection):
 @dataclass
 class FalCredentials(Credentials):
     default_environment: str = "local"
-
     teleport: Optional[TeleportCredentials] = None
+    host: str = ''
+    key_secret: str = ''
+    key_id: str = ''
```

Comment:
Maybe do we want to do an object in a single property instead of 3 separate properties so we know they are ALL in there. 

Something like
```
my_profile:
  target: fal_cloud
  outputs:
    fal_cloud:
      type: fal
      db_profile: db
      cloud: 
        host: my_host
        key_id: my_id
        key_secret: my_secret
    db:
      type: postgres
```

Reply:
This is a great suggestion! What do you think, @burkaygur ?

Reply:
Another thing is that host can also be there if we're using isolate server but not isolate cloud.

Reply:
In that case we could have another object for isolate server? Or what do you think is a good way to handle this?

Reply:
I think what we have is OK for now. We can ask our early users for feedback.

Reply:
I think we shouldnt indent simply because no other adapter does, so I think its good to stay consistent.

Reply:
The only one I see doing this is [BigQuery](https://docs.getdbt.com/reference/warehouse-setups/bigquery-setup#service-account-json). I think you are right.